utils/syncthing/files/etc/init.d/syncthing

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh /etc/rc.common
+
+START=70
```

Comment:
Use 99 instead unless you have a very good reason

Reply:
Thanks